### PR TITLE
building katello-agent 2.6.0

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Will be pulled in to 3.1

File: katello-agent/katello-agent.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Name: katello-agent
-Version: 2.5.0
+Version: 2.6.0
 Release: 3%{?dist}
```
Comment (stbenjam) [inline katello-agent/katello-agent.spec]: Should this go back down to 1?

Comment (jlsherrill) [inline katello-agent/katello-agent.spec]: Thanks!  50% of the time I remember this everytime

**Comment by stbenjam** [comment]
ACK

**Pull request merged**